api/app/db/models/hfi_calc.py

```diff
@@ -69,8 +69,7 @@ class PlanningWeatherStation(Base):
         {'comment': 'Identifies the unique code used to identify the station'}
     )
 
-    id = Column(Integer, Sequence('planning_weather_station_id_seq'),
-                primary_key=True, nullable=False, index=True)
+    id = Column(Integer, primary_key=True)
```

Comment:
I think losing the named sequence is problematic. Have you checked if alembic is going to try to generate a migration based off this change? Why are we losing the index here?

Reply:
I have checked, yes. The only reason for this change is because creating rows through the session fails with a sequence error.

Reply:
There is (was?) (i think!) an issue with alembic, if you're not explicit with the sequence, it may create a sequence with a different name, so you end up with multiple sequences.

Reply:
Also - we are defining explicit sequences all over in hfi_calc and weather_models - we haven't had insert issues before.

Reply:
Oh - also check if the sequence is only broken on your local machine. Could have been with upgrades + downgrades you did that you've gone out of sync. You'll need to restore from back and run alembic to be sure this change doesn't trigger a migration. I'm pretty sure we need that explicit sequence though.

Reply:
Ok - having researched this - I take it all back. I have no idea why we're using explicit sequence naming.